src/env_cli_vars.nim

```diff
@@ -261,12 +263,12 @@ type StatusDesktopConfig = object
     name: "USE_MOCKED_KEYCARD"
     abbr: "use-mocked-keycard" .}: bool
   httpApiEnabled* {.
-    defaultValue: true
+    defaultValue: DEFAULT_FLAG_CONNECTOR_ENABLED
```

Comment:
This should get the _flag_ value, not the default value of that flag :)

Reply:
Fixed